release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -308,6 +327,20 @@ With this release, the Performance Addon Operator (PAO) must-gather image is no
 
 For further information about gathering debugging information relating to low-latency tuning, see xref:../scalability_and_performance/cnf-low-latency-tuning.adoc#cnf-collecting-low-latency-tuning-debugging-data-for-red-hat-support_cnf-master[Collecting low latency tuning debugging data for Red Hat Support].
 
+[id="ocp-4-14-NRO-image-must-gather"]
+==== Collecting data for the NUMA Resources Operator with the `must-gather` image of the operator
+
+In this release, the `must gather` tool has been updated to collect the NUMA-aware scheduling and NUMA Resources Operator data with the NUMA Resources Operator `must-gather` image.
```

Comment:
"..to collect the NUMA-aware scheduling and NUMA Resources Operator data with .."
  I think it's better to have it as:
  ".. to collect the data of the NUMA Resources Operator with the `must-gather` image of the operator, formerly the `must-gather` image of performance-addon-operator was used for this purpose." 
  ^ I added the previous way of getting this data, not sure if that needs to be mentioned here, so leaving it up to you to decide whether or not to add it.